entity-service/src/main/kotlin/com/egm/stellio/entity/model/EntityAccessControl.kt

```diff
@@ -0,0 +1,82 @@
+package com.egm.stellio.entity.model
+
+import com.egm.stellio.entity.config.Neo4jUriPropertyConverter
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.toNgsiLdFormat
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.springframework.data.annotation.LastModifiedDate
+import org.springframework.data.neo4j.core.convert.ConvertWith
+import org.springframework.data.neo4j.core.schema.DynamicLabels
+import org.springframework.data.neo4j.core.schema.Id
+import org.springframework.data.neo4j.core.schema.Node
+import java.net.URI
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+
+@Node
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
you also don't need this (this is not something you're receiving and want to check)